### PR TITLE
pin redis-py below v2.10.6

### PR DESCRIPTION
**Pull request by MaxBittker**
caused travis regression

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: setup.py [hunk 1]
```diff
@@ -125,7 +125,7 @@
     'python-openid>=2.2',
     'PyYAML>=3.11,<3.12',
     'raven>=5.29.0,<6.0.0',
-    'redis>=2.10.3,<2.11.0',
+    'redis==2.10.5',
```
Comment (mattrobenolt) [inline setup.py]: Just do redis>=2.10.3,<2.10.6

**Review by mattrobenolt — APPROVED** [review]
Just make sure to update the PR title now and squash the commits.

**Comment by mattrobenolt** [comment]
Probably also worth adding information into the PR on why this needed to be done?

**Comment by MaxBittker** [comment]
tbh i'm not sure what it was here that broke it: https://github.com/andymccurdy/redis-py/commit/d48c46d3f8c6943c7995bffbecb0fb4c66daabe2#diff-e4eb329834da3d36278b1b7d943b3bc9R2

**Pull request merged**